pkg/scheduler/apis/config/validation/validation.go

```diff
@@ -88,10 +88,9 @@ func ValidateKubeSchedulerConfiguration(cc *config.KubeSchedulerConfiguration) u
 			}
 		}
 	}
-	if cc.PercentageOfNodesToScore < 0 || cc.PercentageOfNodesToScore > 100 {
-		errs = append(errs, field.Invalid(field.NewPath("percentageOfNodesToScore"),
-			cc.PercentageOfNodesToScore, "not in valid range [0-100]"))
-	}
+
+	errs = append(errs, validatePercentageOfNodesToScore(cc.PercentageOfNodesToScore)...)
```

Comment:
Ref https://github.com/kubernetes/kubernetes/pull/112521#discussion_r977201934, this is incorrect:

```
errs = append(errs, validatePercentageOfNodesToScore(field.NewPath("percentageOfNodesToScore"), cc.PercentageOfNodesToScore))
```

Reply:
Add `path` to `validatePercentageOfNodesToScore` as the first argument.